nextflow/variant_load.nf

```diff
@@ -22,22 +30,87 @@ params.help = null
 // Show help message
 if (params.help) exit 0, helpMessage()
 
-// Test input files
-if (!params.variant_load_props || !params.eva_pipeline_props) {
-    if (!params.variant_load_props) log.warn('Provide a variant load properties file using --variant_load_props')
+// Test inputs
+if (!params.valid_vcfs || !params.project_accession || !params.load_job_props || !params.eva_pipeline_props || !params.logs_dir) {
+    if (!params.valid_vcfs) log.warn('Provide validated vcfs using --valid_vcfs')
+    if (!params.project_accession) log.warn('Provide project accession using --project_accession')
+    if (!params.load_job_props) log.warn('Provide job-specific properties using --load_job_props')
     if (!params.eva_pipeline_props) log.warn('Provide an EVA Pipeline properties file using --eva_pipeline_props')
+    if (!params.logs_dir) log.warn('Provide logs directory using --logs_dir')
     exit 1, helpMessage()
 }
 
-variant_load_props = Channel.fromPath(params.variant_load_props)
+// Valid vcfs are redirected to merge step or directly to load
+// See https://nextflow-io.github.io/patterns/index.html#_skip_process_execution
+(vcfs_to_merge, unmerged_vcfs) = (
+    params.needs_merge
+    ? [Channel.from(params.valid_vcfs), Channel.empty()]
```

Comment:
Shouldn't this be fromPath?

Reply:
I think this is okay as is: when we merge, `params.valid_vcfs` is only used to create a file list using [`collectFile`](https://www.nextflow.io/docs/latest/operator.html#collectfile), so they need to be passed as strings rather than files.